### PR TITLE
add lxd-profile-subordinate charm

### PR DESCRIPTION
**Pull request by SimonRichardson**
## Description of change

Adding a lxd-profile-subordinate charm to facilitate and test the
lxd profile updates by charms.

## QA steps

deploy the charm locally.

### CODE REVIEW

**Comment by SimonRichardson** [comment]
https://trello.com/c/ScqbJXnO/490-create-a-sample-subordinate-charm-with-a-second-lxd-profile-tweak-in-the-charm

File: testcharms/charm-repo/quantal/lxd-profile-subordinate/README [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Overview
+
+Defines a subordinate for the LXD profile.
+
+# Usaage
+
+juju deploy lxd-profile
+juju add-relation lxd-profile-subordinate lxd-profile
```
Comment (hmlanigan) [inline testcharms/charm-repo/quantal/lxd-profile-subordinate/README]: juju deploy lxd-profile-subordinate

File: testcharms/charm-repo/quantal/lxd-profile-subordinate/lxd-profile.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+# blacklisted config
+#
+# boot.autostart: "true"
+# limits.... 
+# migration... 
+devices:
+#
+# allowed devices
+#
+  sandisk:
```
Comment (hmlanigan) [inline testcharms/charm-repo/quantal/lxd-profile-subordinate/lxd-profile.yaml]: It'd be helpful to have comments on the keys which will provide something new to the machines lxd profile and which we expect to supersede keys in the lxd-profile charm.

**Comment by jameinel** [comment]
$$merge$$

**Comment by SimonRichardson** [comment]
$$merge$$

**Comment by SimonRichardson** [comment]
$$merge$$

**Pull request merged**